internal/config.go

```diff
@@ -143,3 +161,21 @@ func DefautConfigPath() string {
 	}
 	return filepath.Join(homeDirectoryPath, ".local/share/hz-cli", defaultConfigFilename)
 }
+
+func UpdateConfigWithSSL(config *hazelcast.Config, sslc *SSLConfig) error {
+	if sslc.ServerName != "" && sslc.InsecureSkipVerify {
+		return fmt.Errorf("SSL.ServerName and SSL.InsecureSkipVerify are mutually exclusive")
```

Comment:
```suggestion
		return fmt.Errorf("SSL.ServerName and SSL.InsecureSkipVerify are mutually exclusive\n")
```

Reply:
I think we shouldn't put formatting code in the error messages. Is `\n` necessary for other error messages as well?

Reply:
My bad, I commented on wrong place